twitter/tweepy_scraper.py

```diff
@@ -41,6 +41,42 @@ def on_error(self, status_code):
             #returning False in on_data disconnects the stream
             return False
 
+
+def parse_status(status):
+    '''
+    Parse through the status and only keep the fields that are relevant to the model.
+    This will save hardware space.
+    :param status:
+    :return: a dictionary of all of the fields that we decided to keep
+    '''
+    user_name = status.user.screen_name
+    users_loc = status.user.location
+    tweet_coords = status.coordinates
+    text = status.text
+    # symbols_in_text = list(status.entities.hastags.symbols.text)
+    # hashtags_in_text = list(status.entities.hastags.text)
+    created_date = status.created_at
+
+    # TODO: Implement get_score
+    # lead_score = get_score(text)
+
+    table_fields = {
+        "user_name": user_name,
+        "user_location": users_loc,
+        "tweet_coordinates": tweet_coords,
+        "text": text,
+        # "symbols_in_text": symbols_in_text,
+        # "hashtags": hashtags_in_text,
+        "created_date": created_date,
+        #"lead_score": lead_score,
+    }
+
+
+    return table_fields
+
+
+
+
 #Start streaming the data.
 # stream_listener = StreamListener()
 # stream = Stream(auth=api.auth, listener=stream_listener)
```

Comment:
Too many blank spaces here. There should generally never be more than 2 blank lines. Two lines before a function or class definition, and one blank line everywhere else (including method definitions within a class). https://www.python.org/dev/peps/pep-0008/#blank-lines